src/Entity/Address.php

```diff
@@ -8,8 +8,8 @@
  * @ORM\Entity(repositoryClass="EnMarche\MailerBundle\Repository\AddressRepository")
  * @ORM\Table(
  *     name="mailer_addresses",
- *     indexes={
- *         @ORM\Index(name="unique_address_idx", columns={"canonical_email", "name"})
+ *     uniqueConstraints={
+ *         @ORM\UniqueConstraint(name="unique_email_name_idx", columns={"canonical_email", "name"})
```

Comment:
Why the `_idx` suffix for the constraint name ? can you remove it ?

Reply:
@thomasbisignani done